datasets/german_common_crawl/german_common_crawl.py

```diff
@@ -0,0 +1,143 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""German Common Crawl"""
+
+from __future__ import absolute_import, division, print_function
+
+import datasets
+
+
+# Find for instance the citation on arxiv or on the dataset repo/website
+_CITATION = """\
+@inproceedings{wenzek2020ccnet,
+  title={CCNet: Extracting High Quality Monolingual Datasets from Web Crawl Data},
+  author={Wenzek, Guillaume and Lachaux, Marie-Anne and Conneau, Alexis and Chaudhary, Vishrav and Guzm{\'a}n, Francisco and Joulin, Armand and Grave, {\'E}douard},
+  booktitle={Proceedings of The 12th Language Resources and Evaluation Conference},
+  pages={4003--4012},
+  year={2020}
+}
+"""
+
+_DESCRIPTION = """\
+German Extract from Common Crawl (High Quality only)
+This Dataset is for pretraining a German Language Model (Unsupervised) or tune a Multilingual Model specifically to German"""
+
+
+_URL = [
+    "https://s3.amazonaws.com/datasets.huggingface.co/datasets/datasets/german-nlp-group/german_common_crawl/de_head_0000_2015-48.tar.gz",
+    "https://s3.amazonaws.com/datasets.huggingface.co/datasets/datasets/german-nlp-group/german_common_crawl/de_head_0000_2016-18.tar.gz",
+]
```

Comment:
Are there only two parts of 2GB ? Not all the 200GB of data ?

Reply:
@thomwolf actually wanted to put this PR on hold until there is a solution for hosting very large datasets at HF. You can see parts our discussions in the datasets-external slack channel

Reply:
Ok right !